src/components/Header/Header.js

```diff
@@ -30,17 +38,17 @@ function Header() {
             </Link>
           </li>
           <li className={`nav-item dropdown ${show ? 'show' : ''}`}>
-            <Link
+            <div
```

Comment:
any reason why you are changing this to div? the to attribute will not work here

Reply:
It was giving me problems early on but I had those resolved. Will be changing it back.